upgrade/upgrade-3.2.2.php

```diff
@@ -1,39 +0,0 @@
-<?php
-/**
- * 2007-2019 PrestaShop.
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
- * that is bundled with this package in the file LICENSE.txt.
- * It is also available through the world-wide-web at this URL:
- * https://opensource.org/licenses/AFL-3.0
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to http://www.prestashop.com for more information.
- *
- * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2019 PrestaShop SA
- * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
- * International Registered Trademark & Property of PrestaShop SA
- */
-if (!defined('_PS_VERSION_')) {
-    exit;
-}
-
-function upgrade_module_3_2_2(Ps_Facetedsearch $module)
```

Comment:
Yes we need this, and as I said here: https://github.com/PrestaShop/ps_facetedsearch/pull/133#discussion_r321442805
Since it's a new feature you can name it upgrade_module_3_3_0